hassio/system/hassio-supervisor-info.html

```diff
@@ -36,13 +37,16 @@
     <paper-card heading="Hass.io supervisor">
       <div class="card-content">
         <table class='info'>
+          <template is='dom-if' if='[[data.beta_channel]]'>
```

Comment:
I think that this is less important than the version, let's move it to the row below.